spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/cql/Schema.scala

```diff
@@ -138,7 +150,8 @@ case class TableDef(
     clusteringColumns: Seq[ColumnDef],
     regularColumns: Seq[ColumnDef],
     indexes: Seq[IndexDef] = Seq.empty,
-    isView: Boolean = false) extends StructDef {
+    isView: Boolean = false,
+    options: String = "") extends StructDef {
```

Comment:
I still don't like being able to just pass a string here, If you really think we need this I think it should at least be Seq[String] and we should just require that they not contain "AND" or "WITH". Then we can convert the append code below into  a string join instead of having the more complicated logic.


This removes the need for the "appendOptions" function and replaces it with
```
require(options.forAll( option => !(option.toLowerCase.contains("and") && !(option.toLowerCase.contains("WITH")), "Table options must not contain "WITH OR AND"
(options +: clusteringOptions).mkString("WITH", "AND")```
```